src/kfactory/kcell.py

```diff
@@ -2967,6 +2977,22 @@ def ysize(self) -> float:
         """Returns the height of the bounding box."""
         return self.parent._kdb_cell.dbbox().height()
 
+    @property
+    def x(self) -> float:
+        """X coordinate of the port in um."""
+        return self.parent._kdb_cell.dbbox().center().x
+
+    @property
+    def y(self) -> float:
+        """Y coordinate of the port in um."""
+        return self.parent._kdb_cell.dbbox().center().y
+
+    @property
+    def center(self) -> tuple[float, float]:
+        """Coordinate of the port in um."""
+        vec = self.parent._kdb_cell.dbbox().center()
+        return (vec.x, vec.y)
```

Comment:
to keep this consistent with other classes, just return the DPoint instead of the tuple, this is what we did in other cases. We could theoretically rethink this wrt the new Polygon handling, but I think return a point is safer, as it indicates the relation to dbu as well